### DISCUSSION TITLE
Use AFFiNE to manage GIthub Issues and Discussions​

### DISCUSSION THREAD

**Discussion by omarcostahamido** [toeverything/AFFiNE#6174] [Ideas]
### Description

Would be nice to be able to integrate affine with github issues and disucussions, to create a kanban or list database, for example

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by EYHN** [comment]
Great advice, I'm currently trying  use GitHub project to do this.

**Comment by omarcostahamido** [comment]
sorry @EYHN why did you close it? Is it possible to do already? Or have already decided that you are not implementing it?

**Comment by EYHN** [comment]
Sorry, my mistake, I think I misunderstood, if this is a feature request, do you mean using affine to manage github issues and discussions?

I imagine this will be supported via a plugin in the future.

We won't be able to implement the plugin system in the short term, but I'd like to hear the community's opinion on this, let me reopen this issue.

**Comment by omarcostahamido** [comment]
Yes, i used the feature request template.
I mean, similar to notion, to have a database/table in Affine linked to the list of issues on github and its status.
Doesn't have to be full remote management of the issues, just listing.
Also similar to what you are already doing on your website affine.pro, which looks really nice btw (what are you using to achieve it there?).

Best
